### DISCUSSION TITLE
How to stop temporarily Eruption-GUI ?

### DISCUSSION THREAD

**Discussion by tomjedusor** [X3n0m0rph59/eruption#163] [Q&A]
Firstly thank you very much for this software which works wonderfully and allows marvellous effect.
The question is in the title, I would just like to know (to help OpenRGB dev) how to stop Eruption and restart it?
And if it's a "make stop" from where?

**Comment by tomjedusor** [comment]
I have found this to type in the terminal :

`systemctl --user disable --now eruption-audio-proxy.service
systemctl --user disable --now eruption-process-monitor.service

sudo systemctl disable --now eruption.service`

**Comment by X3n0m0rph59** [comment]
Hi @tomjedusor, thank you for posting this question!

To fully disable Eruption and prohibit automatic starting of the daemon please run the following commands:

```shell
systemctl --user disable --now eruption-audio-proxy.service
systemctl --user disable --now eruption-process-monitor.service

sudo systemctl mask eruption.service && sudo systemctl stop eruption.service
```

to re-enable Eruption, please run:

```shell
sudo systemctl unmask eruption.service && sudo systemctl restart eruption.service

systemctl --user enable --now eruption-audio-proxy.service
systemctl --user enable --now eruption-process-monitor.service
```
